roles/rook/README.rst

```diff
@@ -0,0 +1,107 @@
+Rook
+====
+
+This role is used for updating and recovering the rook jenkins in the rook ci vpc.
+
+Those are the functions in this role are
+
+**rook-jenkins-update**  For updating rook jenkins version to the version defined in the "jenkins_master_image" varible
+
+**rook-os-update**  For updating rook jenkins OS packages
+
+**rook-recovery**  For recoverying the Prod-jenkins instance from the image defined in the "image" varible in a case that the instance was deleted or crashed
```

Comment:
Sorry, can you make these bullet points and either add a colon or " - " between the title and description?